lib/travis/build/addons/apt.rb

```diff
@@ -126,7 +126,7 @@ def add_apt_packages
 
             unless whitelisted.empty?
               sh.export 'DEBIAN_FRONTEND', 'noninteractive', echo: true
-              sh.cmd "sudo -E apt-get -yq update &>> ~/apt-get-update.log", echo: true, timing: true
+              sh.cmd "sudo -E apt-get -yq update", echo: true, timing: true
```

Comment:
Does `apt-get update` return non-zero if there is an error? If so, it might be better to dump the logs only when there is a problem.